cli/tests/it/config.rs

```diff
@@ -98,6 +98,7 @@ forgetest!(can_extract_config_values, |prj: TestProject, mut cmd: TestCommand| {
         build_info_path: None,
         fmt: Default::default(),
         __non_exhaustive: (),
+        __warnings: vec![],
```

Comment:
I'm ok with naming this just `warnings`

Reply:
we can if you want, but I think I like the naming convention of using a double underscore unless its an actual configuration option

Reply:
that makes sense, let's keep it then